docs/sdk/wrapped-keys/overview.md

```diff
@@ -60,12 +40,146 @@ yarn add @lit-protocol/wrapped-keys
 </TabItem>
 </Tabs>
 
-With the SDK, you have the following functions available to you:
+## How it Works
+
+Currently the available actions of the Wrapped Keys SDK are:
+
+- [Generating a Wrapped Key](./generating-wrapped-key.md)
+- [Importing a private key as a Wrapped Key](./importing-key.md)
+- [Exporting a Wrapped Key](./exporting-wrapped-key.md)
+- [Getting Wrapped Key Metadata](./getting-wrapped-key-metadata.md)
+- [Storing Wrapped Key Metadata](./storing-wrapped-key-metadata.md)
+- [Listing all Wrapped Keys associated with a PKP](./listing-wrapped-keys.md)
+- [Signing an arbitrary message with a Wrapped Key](./sign-message.md)
+- [Signing an Ethereum or Solana transaction with a Wrapped Key](./sign-transaction.md)
```

Comment:
is it just ETH and SOL? Or should we say 'blockchain tx' and then include these two examples? (as well as an example showing how you could do this for any chain)

Reply:
Well the SDK specifically supports signing ETH and SOL txs. Users would need to implement their own Lit Actions to perform signatures for other transaction types/signature schemes

Reply:
Changed it to:

> - [Signing a blockchain transaction with a Wrapped Key](./sign-transaction.md)
>     - Currently Ethereum and Solana transactions are supported out-of-the-box using the Wrapped Keys SDK, however, you can implement your own custom Wrapped Keys Lit Action to support other transaction types and/or signature schemes. For more information on how to do this, please refer to [this guide](./custom-wrapped-keys.md).